peer-pod-controller/controllers/peerpodconfig_webhook.go

```diff
@@ -0,0 +1,57 @@
+package controllers
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"net/http"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type PodAnnotator struct {
```

Comment:
We have the webhook already available separately. Wouldn't the webhook here be a duplicate ?

Reply:
you are right, this is a leftover that I can remove

Reply:
removed it